### PR TITLE
Use the default String serializer and deserializer if not set

### LINKED ISSUE #8799: RM: Consider defaulting key and value serializer and deserializer
These currently need to be defined on every channel with config like this:

```
mp.messaging.incoming.foo.key.deserializer=org.apache.kafka.common.serialization.StringDeserializer
mp.messaging.incoming.foo.value.deserializer=org.apache.kafka.common.serialization.StringDeserializer
```

These long keys and values are a pain to enter manually. We should consider defaulting them to the `String` serializer and deserializer as we think that's a common use case.

### PR DESCRIPTION
**Pull request by tevans78**
#build 
fixes #8799

### CODE REVIEW

File: dev/com.ibm.ws.microprofile.reactive.messaging.kafka/src/com/ibm/ws/microprofile/reactive/messaging/kafka/KafkaIncomingConnector.java [hunk 1]
```diff
@@ -98,6 +98,10 @@ private void shutdown() {
 
         // Configure our defaults
         Map<String, Object> consumerConfig = new HashMap<>();
+
+        consumerConfig.putIfAbsent(KafkaConnectorConstants.KEY_DESERIALIZER, KafkaConnectorConstants.DEFAULT_DESERIALIZER);
```
Comment (Azquelt) [inline dev/com.ibm.ws.microprofile.reactive.messaging.kafka/src/com/ibm/ws/microprofile/reactive/messaging/kafka/KafkaIncomingConnector.java]: `putIfAbsent` is unnecessary as the map is created empty on the preceeding line

File: dev/com.ibm.ws.microprofile.reactive.messaging.kafka/src/com/ibm/ws/microprofile/reactive/messaging/kafka/KafkaOutgoingConnector.java [hunk 2]
```diff
@@ -54,6 +54,9 @@
                                                           .filter(k -> !KafkaConnectorConstants.NON_KAFKA_PROPS.contains(k))
                                                           .collect(Collectors.toMap(Function.identity(), (k) -> config.getValue(k, String.class)));
 
+        producerConfig.putIfAbsent(KafkaConnectorConstants.KEY_SERIALIZER, KafkaConnectorConstants.DEFAULT_SERIALIZER);
```
Comment (Azquelt) [inline dev/com.ibm.ws.microprofile.reactive.messaging.kafka/src/com/ibm/ws/microprofile/reactive/messaging/kafka/KafkaOutgoingConnector.java]: I don't like modifying the map returned from `collect()`. I don't think there's any guarantee in the API that it's mutable.

Comment (Azquelt) [reply dev/com.ibm.ws.microprofile.reactive.messaging.kafka/src/com/ibm/ws/microprofile/reactive/messaging/kafka/KafkaOutgoingConnector.java]: I'd prefer it if this class used the same structure as `KafkaIncomingConnector` for handling config defaults.

**Comment by tevans78** [comment]
#build

**Comment by tevans78** [comment]
This PR is now redundant. Closing.

**Pull request closed without merging**